### DISCUSSION TITLE
Use custom headers

### DISCUSSION THREAD

**Discussion by jeremyzaire** [DigitalPebble/storm-crawler#908] [Ideas]
Hi ! 
I tried to use com.digitalpebble.stormcrawler.protocol.okhttp.HttpProtocol to scrape some websites but I can't add custom headers. Is there a way to add custom headers without write a new Protocol ?

Thanks regards,
Jeremy

**Comment by jnioche** [comment]
thanks @jeremyzaire 
I thought this was the case but looking at the code I can't see any way of doing this. That would be a great contribution if you are interested in doing it.

**Comment by jnioche** [comment]
@jeremyzaire any chance you could try #912?

**Reply by jeremyzaire** [reply]
Thanks @jnioche I will test today. Thanks for your contribution

**Reply by jeremyzaire** [reply]
It works fine ! Thanks you for your amazing job

**Comment by jnioche** [comment]
Fab, thanks for the feedback
